### PR TITLE
renamed getLoop() to isLooping() and setLoop() to setLooping()

### PR DESCRIPTION
**Pull request by pinsvin00**
## Description
This PR is related to the issue #3184
https://github.com/SFML/SFML/issues/3184

## Tasks

-   [ ] Tested on Linux
-   [X] Tested on Windows
-   [ ] Tested on macOS
-   [ ] Tested on iOS
-   [ ] Tested on Android

## How to test this PR?
Some usage of setLooping() is already present in SoundEffects.cpp example project. Please check this example project.

### CODE REVIEW

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Comment by ChrisThrasher** [comment]
Looks like you need to update the unit tests. If you build SFML with the `dev` preset then CMake will automatically enable all the extra developer tools including the tests. I'd recommend using that preset any time you want to work on the library.

**Comment by pinsvin00** [comment]
Thanks for noticing, I had some issues with cmake so I missed all of the tests.

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

File: include/SFML/Audio/Music.hpp [hunk 1]
```diff
@@ -306,7 +306,7 @@ class SFML_AUDIO_API Music : public SoundStream
 /// music.setPosition({0, 1, 10}); // change its 3D position
 /// music.setPitch(2);             // increase the pitch
 /// music.setVolume(50);           // reduce the volume
-/// music.setLoop(true);           // make it loop
+/// music.setLooping(true);           // make it loop
```
Comment (ChrisThrasher) [inline include/SFML/Audio/Music.hpp:309]:
```suggestion
/// music.setLooping(true);        // make it loop
```

**Review by ChrisThrasher — CHANGES_REQUESTED** [review]

**Review by ChrisThrasher — APPROVED** [review]

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Review by ChrisThrasher — APPROVED** [review]

**Comment by eXpl0it3r** [comment]
Thank you! 😊

**Pull request merged**